src/_includes/macros/util/common-form-fields.html

```diff
@@ -0,0 +1,134 @@
+                'Kansas',
+                'Kentucky',
+                'Louisiana',
+                'Maine',
+                'Maryland',
+                'Massachusetts',
+                'Michigan',
+                'Minnesota',
+                'Mississippi',
+                'Missouri',
+                'Montana',
+                'Nebraska',
+                'Nevada',
+                'New Hampshire',
+                'New Jersey',
+                'New Mexico',
+                'New York',
+                'North Carolina',
+                'North Dakota',
+                'Ohio',
+                'Oklahoma',
+                'Oregon',
+                'Pennsylvania',
+                'Rhode Island',
+                'South Carolina',
+                'South Dakota',
+                'Tennessee',
+                'Texas',
+                'Utah',
+                'Vermont',
+                'Virginia',
+                'Washington',
+                'West Virginia',
+                'Wisconsin',
+                'Wyoming'
+              ],
+              size,
+              required,
+              disabled) }}
+{% endmacro %}
+
+
+{# ==========================================================================
+
+   common_form_fields.date_picker()
+
+   ==========================================================================
+
+   Description: Returns a date picker
+
+   legend:      Name of the field.
+
+   required:    Whether the field is required. Defaults to false.
+
+   disabled:    Whether the field is disabled. Defaults to false.
+
+   ========================================================================== #}
+
+{% macro date_picker(legend='Enter Date Range', required=false, disabled=false) %}
+    <fieldset class="u-reset">
+        <div class="form-l_col
+                    form-l_col-1
+                    u-mb0">
+            <legend class="form-label-header">
+                {{ legend }}
+            </legend>
+        </div>
+        <div class="form-l_col form-l_col-1-2">
+            <label class="micro-copy micro-copy__large"
+                   for="form_date-picker-starting-date">
+                From:
+            </label>
+            <input class="input__long"
+                   id="form_date-picker-starting-date"
+                   placeholder="mm/dd/yyyy"
+                   type="text"
+                   {{ 'required' if required }}
+                   {{ 'disabled' if disabled }}>
+        </div>
+        <div class="form-l_col
+                    form-l_col-1-2
+                    input__long">
+            <label class="micro-copy micro-copy__large"
+                   for="form_date-picker-ending-date">
+                To:
+            </label>
+            <input class="input__long"
+                   id="form_date-picker-ending-date"
+                   placeholder="mm/dd/yyyy"
+                   type="text"
+                   {%- if required -%}required{%- endif -%}
```

Comment:
Should be updated to be like https://github.com/cfpb/cfgov-refresh/pull/965/files#diff-1823c209048b4eea154322ba509b3e51R116 This and L:131